packages/shared/chat/sagas.ts

```diff
@@ -119,16 +121,27 @@ function* handleSendMessage(action: SendMessage) {
     // If the message was not a command ("/cmdname"), then send message through wire
     const currentUserId = yield select(getCurrentUserId)
     if (!currentUserId) throw new Error('cannotGetCurrentUser')
+    if (isChannel) {
+      entry = {
+        messageType: ChatMessageType.PUBLIC,
+        messageId: uuid(),
+        sender: currentUserId,
+        recipient,
+        body: message,
+        timestamp: Date.now()
+      }
+      store.dispatch(sendChannelMessage(recipient, message))
```

Comment:
This is illegal, should have failed in tests.

```suggestion
      yield put(sendChannelMessage(recipient, message))
```